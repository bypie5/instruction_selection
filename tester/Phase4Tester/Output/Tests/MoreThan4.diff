--- SelfTestCases/MoreThan4.out	2011-04-28 14:55:21.000000000 -0700
+++ ./Output/Tests/MoreThan4.out	2020-03-13 21:48:04.000000000 -0700
@@ -1,14 +1,6 @@
-1
-2
-3
-4
-5
-6
-6
-5
-4
-3
-2
-1
-0
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/MoreThan4.compiler.out line 66 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/MoreThan4.compiler.out line 68 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/MoreThan4.compiler.out line 70 column 1: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/MoreThan4.compiler.out line 2 column 1: Symbol "vmt_MT4" not found in symbol table.
 
+Processing terminated due to errors.
