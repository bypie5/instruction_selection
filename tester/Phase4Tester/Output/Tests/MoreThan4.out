1
2
3
0
0
0
0
0
0
2147479528
4194608
3
0

